### DISCUSSION TITLE
OTP expiration – Invite user by email

### DISCUSSION THREAD

**Discussion by msaaddev** [supabase/supabase#27224] [Questions]
We are inviting users by emails using supabase `inviteUserByEmail` method. We have setup the `/confirm` route in our app according to the documentation.

The problem is some of the users end up getting an error. We are assuming that since the invitation works for most of the users, the OTP expires for the rest. 

1. My question is how long the invite OTP is valid? I tried to find it in the docs but couldn't.
2. Can we increase this duration? If yes, how?

**Comment by jason-curtis** [comment]
What error message are they getting?
Just now I've been getting 504 timeout errors on the https://<my app ID>.supabase.co/auth/v1/otp endpoint, so there may be some kind of system issue that's affecting both of us.

**Comment by GaryAustin1** [comment]
Could it be the link is being used by their email service?
https://supabase.com/docs/guides/auth/auth-email-templates#email-prefetching
Otherwise I believe this is the setting....
![image](https://github.com/supabase/supabase/assets/54564956/f81381f2-0d72-4c39-a265-cb64f96e5d5b)
